### PR TITLE
[CAS-995] Allow customizing ChannelListView via theme

### PR DESCRIPTION
**Pull request by adasiewiczr**
https://stream-io.atlassian.net/browse/CAS-995
### 🎯 Goal

1. Add missing attrs to `ChannelListView`.
2. Allow customizing `ChannelListView `via theme

### 🧪 Testing

Try to apply the `ChannelListView` theme.

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [x] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [ ] Comparison screenshots added for visual changes
- [ ] Affected documentation updated (docusaurus, tutorial, CMS (task created))
- [x] Reviewers added

### CODE REVIEW

File: stream-chat-android-ui-components/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -337,6 +337,48 @@
         <item name="android:layout_marginBottom">12dp</item>
     </style>
 
+    <style name="StreamUi.ChannelListView" parent="StreamUi">
```
Comment (leandroBorgesFerreira) [inline stream-chat-android-ui-components/src/main/res/values/styles.xml:340]: Is this being used? I think it should go inside `StreamUiTheme`.

File: UNRELEASED_CHANGELOG.md [hunk 2]
```diff
@@ -64,6 +64,9 @@
 ### ⬆️ Improved
 
 ### ✅ Added
+- Now you can configure the style of `AttachmentMediaActivity`
+- Added `streamUiLoadingView`, `streamUiEmptyStateView` and `streamUiLoadingMoreView` attributes to `ChannelListView` and `ChannelListViewStyle`
+- Added possibility to customize `ChannelListView` using theme. Check `StreamUi.ChannelListView` style.
```
Comment (leandroBorgesFerreira) [inline UNRELEASED_CHANGELOG.md:69]: The user needs to override `streamUiChannelListViewStyle`. I think it is worth make this clear.

**Review by leandroBorgesFerreira — APPROVED** [review]
Just one comment

File: stream-chat-android-ui-components/src/main/res/values/styles.xml (hunk 1 above)
Comment (adasiewiczr) [reply stream-chat-android-ui-components/src/main/res/values/styles.xml:340]: It's used [here](https://github.com/GetStream/stream-chat-android/pull/2064/files#diff-1d31aa89c0698cf181975aecd03f3f8cd8d448fd58e52d1ae2cd7376aec99391R76) so we don't need to add it to `StreamUiTheme`

**Pull request merged**